batch/data-plane/src/test/java/com/microsoft/azure/batch/FileTests.java

```diff
@@ -42,14 +42,16 @@ public static void cleanup() throws Exception {
     @Test
     public void canReadFromTaskFile() throws Exception {
         // CREATE
-        String jobId = getStringIdWithUserNamePrefix("-Job-" + (new Date()).toString().replace(' ', '-').replace(':', '-').replace('.', '-'));
+        String jobId = getStringIdWithUserNamePrefix("-Job-canReadFromTaskFile");
         String taskId = "mytask";
         int TASK_COMPLETE_TIMEOUT_IN_SECONDS = 60; // 60 seconds timeout
 
         try {
 
             PoolInformation poolInfo = new PoolInformation();
-            poolInfo.withPoolId(livePool.id());
+            if(isRecordMode()) {
```

Comment:
why this can't be set at playback mode?

Reply:
It can, 
updated the code.
Also updated other similar places which I missed before.